database/dynamodb.go

```diff
@@ -1,62 +1,79 @@
 package database
 
 import (
+	"fmt"
 	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/dynamodb"
 	"log"
 	"os"
-	"fmt"
 )
 
-var Constants map[string]string = map[string]string{
-	"ENV":        "ENVIRONMENT",
-	"DEV":        "DEVELOPMENT",
-	"PROD":       "PRODUCTION",
-	"REGION":     "AWS_REGION",
-	"ACCESS_KEY": "AWS_ACCESS_KEY",
-	"SECRET_KEY": "AWS_SECRET_KEY",
-}
-
 type AWSCredentials struct {
 	AccessKey string
 	SecretKey string
 	Region    string
 }
 
+type GetItemResult struct {
+	Item map[string]*dynamodb.AttributeValue
+}
+
 var db *dynamodb.DynamoDB
 
 func init() {
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println(err)
+		}
+	}()
 
-	env, found := os.LookupEnv(Constants["ENV"])
```

Comment:
Have removed this code, as when trying to test with lambda in local, which works on Docker image, the `.env` file was not found, had to store the environment variables in [template.yaml](https://github.com/Real-Dev-Squad/feature-flag-backend/pull/27/files#diff-1363ef5ce8886100842332c97163aad7934237e1fe49b5d40422b45fdc30f38e).

Reply:
An alternate here could be having environment specific yaml files that you name with some convention and then use AWS::Include transform in your template.yaml.